addme

```diff
@@ -0,0 +1,105 @@
+#/bin/bash
+
+#set -x 
+
+OS=`uname -s`
+mygid=7903
+myuid="556"
+myuname="jeffcof"
+mygrpname=scxdev
+
+while [ $# -ne 0 ]; do
+	case "$1" in 
+		-uid )
+			myuid=$2
+			shift
+		;;
+
+		-uname )
+			myuname=$2
+			shift
+		;;
+
+		* )
+			echo "addme -uid <uid> -uname <username>"
+			exit 2
+		;;
+
+	esac
+
+	shift
+
+done
+
+myhome=/home/$myuname
+
+if [ "$myuid" = "<UID GOES HERE>" -o "$myuname" = "<USERNAME GOES HERE>" ]; then
+	echo "You need to modify the script with your information"
+	echo "or supply the information on the command line..."
+	exit 1
+fi
+
+# Override default for Solaris
+if [ "$OS" = "SunOS" ]; then
+	myhome=/export/home/$myuname
+fi
```

Comment:
Why force home directory?  Useradd will automatically create a home directory.  It doesn't matter where.

Reply:
In SunOS, the user home directory is /export/home/<username>